cpp/tests/transform/integration/assert-unary.h

```diff
@@ -16,47 +13,45 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#ifndef GDF_TESTS_UNARY_OPERATION_INTEGRATION_ASSERT_BINOPS_H
-#define GDF_TESTS_UNARY_OPERATION_INTEGRATION_ASSERT_BINOPS_H
 
-#include <tests/utilities/legacy/column_wrapper.cuh>
-#include <tests/utilities/legacy/scalar_wrapper.cuh>
+#pragma once
+
+#include <tests/utilities/column_wrapper.hpp>
+#include <tests/utilities/column_utilities.hpp>
 #include <gtest/gtest.h>
 
 namespace cudf {
 namespace test {
 namespace transformation {
 
 template <typename TypeOut, typename TypeIn, typename TypeOpe>
-void ASSERT_UNARY(cudf::test::column_wrapper<TypeOut>& out,
-                  cudf::test::column_wrapper<TypeIn>& in,
+void ASSERT_UNARY(std::unique_ptr<column>& out,
```

Comment:
This looks like it could/should be just a `column_view`. 
```suggestion
void ASSERT_UNARY(column_view const& out,
```